File: app/src/main/java/mariavv/fitnesspal/presentation/handbook/HandBookView.java
package mariavv.fitnesspal.presentation.handbook;

import android.database.Cursor;

import com.arellomobile.mvp.MvpView;
import com.arellomobile.mvp.viewstate.strategy.AddToEndSingleStrategy;
import com.arellomobile.mvp.viewstate.strategy.StateStrategyType;

@StateStrategyType(AddToEndSingleStrategy.class)
interface HandBookView extends MvpView {

    void updateFeed(Cursor c);

    void setTitle();
}
//import com.arellomobile.mvp.viewstate.strategy.SingleStateStrategy;

File: app/src/main/java/mariavv/fitnesspal/presentation/handbook/HandBookView.kt
package mariavv.fitnesspal.presentation.handbook

import android.database.Cursor
import android.support.annotation.StringRes
import com.arellomobile.mvp.MvpView
import com.arellomobile.mvp.viewstate.strategy.AddToEndSingleStrategy
import com.arellomobile.mvp.viewstate.strategy.StateStrategyType

@StateStrategyType(AddToEndSingleStrategy::class)
interface HandBookView : MvpView {

    fun updateFeed(c: Cursor)

    fun setTitle(@StringRes title: Int)
}